github-sync/sync.rb

```diff
@@ -69,6 +69,10 @@ def github_sync(context, run_one)
     # TODO: Move to being based on the async code
     sync_issue_comments(context, sync_db)
   end
+  if(not(run_one) or run_one=='github-sync/pr-reviews')
```

Comment:
Mental note that I'll also need to implement the async side of this.

Reply:
Happy to help here - it will take me a while to figure things out though: My Ruby experience is pretty much non-existent ;)